src/models/receipt.ts

```diff
@@ -0,0 +1,26 @@
+export type aReceipt = {
+  id: string;
+  amount: number;
+  date: string;
+  isTest: boolean;
+  token: string;
+};
+export class Receipt {
```

Comment:
Is there a reason you have the separate `type aReceipt` and the `class Receipt`? It seems like you should be able to just have the `type` since the class is just a container for the data and doesn't have any operations on it.

Reply:
hmm, in the constructor of Receipt class, TS is asking to remove implicit any. what's the best way to remove this?

<img width="519" alt="image" src="https://github.com/user-attachments/assets/a6279583-8319-4d53-9af8-ee16bb5407ff" />

Reply:
ah ok, leaving it as internal since that is where it really is only used. other references point to the class - [commit here](https://github.com/internetarchive/iaux-monthly-giving-circle/pull/15/commits/d2c8179f2efa5fbd686b6969813ff18beaa08164)

Reply:
Well I mean, do you need the class at all since it's just defining the shape of the data?

Where you're doing 
```
const receipt = new Receipt({ 
  amount: 100,
  date: new Date(),
  donor: "boop",
  ...
})
``` 
you can just replace it with
```
const receipt: aReceipt = {
  amount: 100,
  date: new Date(),
  donor: "boop",
  ...
}
```

The type is telling typescript everything it needs to know about the receipt. You really only need an extra class if you need to do extra work on the data internally.